src/ui/popup/theme/controls/theme-control.less

```diff
@@ -1,10 +1,10 @@
 .theme-control {
+    min-height: @size-control-inner;
```

Comment:
Please sort the properties alphabetically (according to [Google CSS guidelines](https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules)).